public/tfWorker.js

```diff
@@ -20,6 +20,7 @@ use.load().then(function (model) {
       encodings = tf.tensor2d(data.encodings)
       // successfully loaded model & downloaded encodings
       isReady = true
+      self.postMessage('ready')
```

Comment:
@ccstan99 I updated the worker to send a message when it's ready, so that I can disable the baseline plaintext search sooner to avoid flickering of search results

the baseline search is there only as a backup if TF doesn't load or something